docs/docs/hooks.md

```diff
@@ -21,15 +21,19 @@ For a full list of hooks, see [Hooks API](/api/hooks).
   beforeCreate(instance, options, fn)
   beforeDestroy(instance, options, fn)
   beforeUpdate(instance, options, fn)
+(5)
+  beforeSave(instance, options, fn)
```

Comment:
What will happen if there is error in `beforeUpdate` should `beforeSave` run in such cases ?